src/wp-admin/includes/upgrade.php

```diff
@@ -2202,6 +2202,20 @@ function upgrade_550() {
 	}
 }
 
+/**
+ * Executes changes made in WordPress 5.6.0.
+ *
+ * @ignore
+ * @since 5.6.0
+ */
+function upgrade_560() {
+
```

Comment:
@aristath I think we should remove this blank line or also add one before the closure :)

Reply:
Good catch, removed :+1: